packages/typescript/test/lit_plugin/BUILD.bazel

```diff
@@ -0,0 +1,36 @@
+# Copyright 2017 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@npm_bazel_typescript//:index.from_src.bzl", "ts_library")
+
+ts_library(
+    name = "lit_plugin",
+    srcs = glob(["*.ts"]),
+    expected_diagnostics = [
+        """TS2322: \\[lit\\] Unknown tag "unknown-element". Did you mean 'lit-element'?""",
+        "TS2322: \\[lit\\] Type '222' is not assignable to 'string",
+        "TS2322: \\[lit\\] Type '333' is not assignable to 'string",
+        """TS2322: \\[lit\\] Type '"aaa"' is not assignable to 'number'""",
+        #"""TS2322: \\[lit\\] Type '"bbb"' is not assignable to 'number'""",
+        """TS2322: \\[lit\\] Type '"ccc"' is not assignable to 'number'""",
+        "TS2322: \\[lit\\] Type '444' is not assignable to 'string",
+        "TS2322: \\[lit\\] Type '{ field: number; }' is not assignable to '{ field: string; }'",
+    ],
+    tsconfig = ":tsconfig.json",
+    deps = [
+        "@npm//lit-element",
+        # FIXME: plugins["@npm//ts-lit-plugin"] instead?
```

Comment:
is it `plugins` is google3?

Reply:
no, in google3 you have `ts_library(use_lit_plugin=True)`
But that's because we are highly locked down for allowing customization. Under Bazel we should probably let people choose their plugins and it's their choice to maintain the build to deal with bad behavior the plugins might cause